imports/plugins/core/components/lib/hoc.js

```diff
@@ -1,8 +1,8 @@
-import _ from "lodash";
+/* eslint-disable node/no-missing-require, node/no-unpublished-require */
```

Comment:
These were some of the new rules added when we did the `recommended` import of `node` lint rules.

We should discuss if we want to keep these rules in our `eslint` config, I know @mikemurray has run across these lint errors a few times too.